HyperV/scripts/EnsureOpenStackServices.ps1

```diff
@@ -0,0 +1,22 @@
+Param(
+    [Parameter(Mandatory=$True)]
+    [string]$serviceUsername,
+    [Parameter(Mandatory=$True)]
+    [string]$servicePassword
+)
+
+$openstackDir = "C:\OpenStack"
```

Comment:
All these variables are used in multiple scripts so please create a config file and store all of them there.